contrib/sacrebleu/sacrebleu.py

```diff
@@ -869,7 +911,7 @@ def main():
                             help='use case-insensitive BLEU (default: actual case)')
     arg_parser.add_argument('--smooth', '-s', choices=['exp', 'floor', 'none'], default='exp',
                             help='smoothing method: exponential decay (default), floor (0 count -> 0.01), or none')
-    arg_parser.add_argument('--tokenize', '-tok', choices=['13a', 'zh'], default='13a',
+    arg_parser.add_argument('--tokenize', '-tok', choices=list(filter(lambda x: x != 'none', tokenizers.keys())), default='13a',
```

Comment:
`choices=[x for x in tokenizers.keys() if x != 'none']`
seems more Pythonic to me

Reply:
All right.